styles/mainStyle.css

```diff
@@ -22,6 +22,7 @@ header {
   height: 9vh;
   min-width: 100vw;
   display: flex;
+  box-shadow: 2px 2px 2px 1px rgba(0, 0, 0, 0.2);
 }
 
```

Comment:
Great design @EvansSnave 🤝

- [ ] [Optional] I would suggest you remove the partners and footer sections from your mobile design of the mobile home page so that your design can replicate the design given.
- Your design
![Your mobile-Home](https://github.com/EvansSnave/capstone-project/assets/63917725/2e89984e-edc4-42a6-90a8-4ca506ea54e4)
- Given design
![Given mobile home](https://github.com/EvansSnave/capstone-project/assets/63917725/9eb27f26-c60d-4a13-b057-3c52c40ed979)